scripts/Run-ReactRedux-Locally.ps1

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env powershell
+#requires -version 4
+
+[CmdletBinding(PositionalBinding = $false)]
+param()
+
+# BEWARE: This script makes changes to source files which you will have to seperate from any changes you want to keep before commiting.
+
+Set-StrictMode -Version 2
+$ErrorActionPreference = 'Stop'
+
+git clean -xdf
+$projects = "$PSScriptRoot/../src/Microsoft.DotNet.Web.Spa.ProjectTemplates"
+
+$csproj = Join-Path $projects "ReactRedux-CSharp.csproj.in"
+(Get-Content $csproj).replace('<PackageReference Include="Microsoft.AspNetCore.App"', '<PackageReference Include="Microsoft.NETCore.App" Version="${MicrosoftNETCoreApp22PackageVersion}" />
```

Comment:
Doesn't this cause warnings by the SDK? This package reference should be implicit. If you need to control Microsoft.NETCore.App, use `<RuntimeFrameworkVersion>`.

Reply:
I've changed the scripts to use the CustomHive instead. It's a more accurate way to test things anyway now that I understand it.